### PR TITLE
fix(ui5-time-picker): accessibility improved

### LINKED ISSUE #3010: TimePicker-based components: Adopt latest screen reader specification
**Is your feature request related to a problem? Please describe.**
At the moment any component that extends TimePickerBase (_ui5-time-picker_ and _ui5-duration-picker_) doesn't have appropriate ARIA support. To fix this, we need to enhance not only their input part, but their sliders as well.

**Describe the solution you'd like**
Input - needs to adopt the [ARIA 1.2 combobox](https://www.w3.org/TR/wai-aria-1.2/#combobox) pattern, which is recommended by the ARIA group:

- `input` element with `type="text"`
- `role="combobox"`
- `aria-haspopup="dialog"`
- `aria-controls="PICKER_ID"`
- `aria-expanded="true"`/`"false"`
- `aria-autocomplete="none"`
- `aria-roledescription="Time Input"` (for _ui5-time-picker)_ / `"Duration Input"` (for _ui5-duration-picker_)

Sliders:
- `ul` element
- `role="listbox"`
- `aria-label="Hours"`/`"Minutes"`/etc.. depending on the slider. Referencing the already existing label (`#ui5-wheelslider-label`) with `aria-labelledby` is also an option.

Slider item:
- `li` element
- `role="option"`
- Indicate selection via `aria-selected="true"`/`"false"`

Feel free to let me know if you require any further information / assistance!

### PR DESCRIPTION
**Pull request by unazko**
The ui5-time-picker and ui5-duration picker inner input
elements and wheel sliders now have aria properties set
according to the screen reader specification.

Fixes #3010

### CODE REVIEW

**Comment by ilhan007** [comment]
it looks ok to me, but I will wait for your internal review

File: packages/main/src/WheelSlider.hbs [hunk 1]
```diff
@@ -19,14 +19,25 @@
 	<div id="{{this._id}}--selection-frame" class="ui5-wheelslider-selection-frame"></div>
 		<div id="{{this._id}}--wrapper" class="ui5-wheelslider-wrapper">
 			{{#if expanded}}
-				<ul id="{{this._id}}--items-list">
+				<ul id="{{this._id}}--items-list" role="listbox" aria-labelledby="{{label}}">
```
Comment (ilhan007) [inline packages/main/src/WheelSlider.hbs:22]: is this {{label}} an ID

**Comment by ilhan007** [comment]
Hi @tsanislavgatev could you provide a review at least from code perspective.

File: packages/main/src/WheelSlider.hbs (hunk 1 above)
Comment (unazko) [reply packages/main/src/WheelSlider.hbs:22]: {{label}} is not an ID. I've mixed the two possible solutions.
I've changed to aria-label instead.

**Comment by tsanislavgatev** [comment]
@ilhan007 @unazko The code looks okay to me.

File: packages/main/src/WheelSlider.hbs (hunk 1 above)
Comment (ilhan007) [inline packages/main/src/WheelSlider.hbs:22]: Still aria-labelledby="{{label}}" is used here line 22 WheelSlider.hbs

**Review by ilhan007 — CHANGES_REQUESTED** [review]

File: packages/main/src/WheelSlider.hbs (hunk 1 above)
Comment (unazko) [reply packages/main/src/WheelSlider.hbs]: There was an if statement with a variation of the ul element. Aria-label is changed in both places now.

**Review by ilhan007 — APPROVED** [review]

**Pull request merged**